internal/trigger/trigger.go

```diff
@@ -24,6 +24,4 @@ import (
 type ITrigger interface {
 	// Initialize performs post creation initializations
```

Comment:
Stutter -- trigger/trigger.go. Might be better to call this trigger/interfaces.go